backend/lib/index.js

```diff
@@ -40,6 +41,7 @@ module.exports = function createApp(config) {
   app.register(organizations, { prefix: "api/organizations" });
   app.register(posts, { prefix: "/api/posts" });
   app.register(users, { prefix: "/api/users" });
+  app.register(feedback, { prefix: "/api/feedback" });
```

Comment:
Minor: the route should be registered in alphabetical order. I guess ESLint didn't catch this for some reason...

Reply:
No worries. That makes sense. I will fix that really quick.

Reply:
Actually, ESLint is currently only checking the "object keys" order, there are additional rules that we can add such ass alphabetic imports... not sure if this works for every part in the code such as calling methods here, but it might help.
For plugins it's probably not always possible since the order might matter (and these routes are added the same as plugins for fastify... didn't find any better way to group the routes)

Reply:
Ah okay that makes sense.